catalog/core/catalog-core-solrcommands/pom.xml

```diff
@@ -65,6 +65,16 @@
             <groupId>org.apache.solr</groupId>
```

Comment:
Should we update the jacoco coverage? Seems like you've added quite a bit of coverage.